packages/nhsuk-frontend/src/nhsuk/components/header/macro-options.mjs

```diff
@@ -1439,6 +1439,47 @@ export const examples = {
     screenshot: {
       viewports: ['desktop']
     }
+  },
+  'where some navigation items are false': {
```

Comment:
Sorry @frankieroberto can we rename this?

Just with the examples ordered A-Z with "blue" or "white" suffixes etc

* blue with navigation
* blue with navigation (justified)
* blue with navigation (overflow)
* blue with navigation (overflow, white)

Makes sense to make this:

```suggestion
  'blue with navigation (empty items)': {
```